vspace/src/main/java/edu/asu/diging/vspace/core/model/display/impl/ExternalLinkDisplay.java

```diff
@@ -0,0 +1,118 @@
+package edu.asu.diging.vspace.core.model.display.impl;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.IExternalLink;
+import edu.asu.diging.vspace.core.model.display.IExternalLinkDisplay;
+import edu.asu.diging.vspace.core.model.impl.ExternalLink;
+import edu.asu.diging.vspace.core.model.impl.VSpaceElement;
+
+public class ExternalLinkDisplay extends VSpaceElement implements IExternalLinkDisplay {
```

Comment:
Let's create a display superclass for all link displays, since they will all have positional data, type data, etc.